src/components/GraphFilter/GraphFind.tsx

```diff
@@ -99,47 +101,67 @@ export class GraphFind extends React.PureComponent<GraphFindProps, GraphFindStat
             <InputGroup>
               <FormControl
                 id="graph_find"
+                name="graph_find"
+                autoComplete="on"
                 type="text"
                 style={{ ...inputWidth }}
                 inputRef={ref => {
                   this.findInputRef = ref;
                 }}
                 onChange={this.updateFind}
-                defaultValue={this.state.findInputValue ? this.state.findInputValue : this.props.findValue}
+                defaultValue={this.state.findInputValue}
                 onKeyPress={this.checkSubmitFind}
                 placeholder="Find..."
               />
               {this.state.findInputValue && (
-                <InputGroup.Button>
-                  <Button onClick={this.clearFind}>
-                    <Icon name="close" type="fa" />
-                  </Button>
-                </InputGroup.Button>
+                <OverlayTrigger
+                  key="ot-clear_find"
```

Comment:
completely trivial - but it is odd that you use "-" and "_" in the key value here. I'm assuming there is a reason. I'm not requesting a change, but I just bring it up because it looks weird.